### PR TITLE
Installing ldap PHP extension

### PR DESCRIPTION
**Pull request by vijayreddy1991**
https://github.com/Shippable/pm/issues/11271

image

ldap for php5.6
image

ldap for php7.0
image

ldap for php7.1
image

ldap for php 7.2
image

### CODE REVIEW

File: version/5_6.sh [hunk 1]
```diff
@@ -22,6 +22,7 @@ curl -sS http://getcomposer.org/installer | php
 chmod +x composer.phar
 mv composer.phar $HOME/.phpenv/versions/5.6/bin/composer
 
+sudo apt-get install php5.6-ldap
```
Comment (ric03uec) [inline version/5_6.sh]: which version finally gets installed? will apt-get not override all previous versions?

Comment (vijayreddy1991) [reply version/5_6.sh]: previously the ldap is not available in php

Comment (ric03uec) [reply version/5_6.sh]: can we print the ldap version for each variation of php. the output is just showing a list of modules and im not sure if the correct version of ldap module is being installed for each php version

Comment (vijayreddy1991) [reply version/5_6.sh]: image

**Pull request merged**